src/draft/NSG/Agent/LegalEntity/NonListedCompany/SignatoryRights.py

```diff
@@ -0,0 +1,408 @@
+        "component parts.",
+        example="Tietotie 4 A 7, 00100 Helsinki, Finland",
+        max_length=250,
+    )
+
+    thoroughfare: Optional[str] = Field(
+        None,
+        title="Thoroughfare",
+        description="The name of a passage or way through from one location to another. A thoroughfare is usually a "
+        "street, but it might be a waterway or some other feature.",
+        example="Avenue des Champs-Élysées",
+        max_length=40,
+    )
+    locator_designator: Optional[str] = Field(
+        None,
+        title="Locator designator",
+        description="A number or sequence of characters that uniquely identifies the locator within the relevant "
+        "scope. In simpler terms, this is the building number, apartment number, etc.",
+        example="Flat 3, 17 or 3 A 4",
+        max_length=10,
+    )
+    locator_name: Optional[str] = Field(
+        None,
+        title="Location name",
+        description="Proper noun(s) applied to the real world entity identified by the locator. The locator name "
+        "could be the name of the property or complex, of the building or part of the building, "
+        "or it could be the name of a room inside a building. The key difference between a locator "
+        "designator and a locator name is that the latter is a proper name and is unlikely to include "
+        "digits.",
+        example="Shumann, Berlaymont building",
+        max_length=40,
+    )
+    address_area: Optional[str] = Field(
+        None,
+        title="Address area",
+        description="The name of a geographic area that groups Addresses. This would typically be part of a city, "
+        "a neighbourhood or village. Address area is not an administrative unit. ",
+        example="Montmartre (in Paris)",
+        max_length=40,
+    )
+    post_code: Optional[str] = Field(
+        None,
+        title="Post code",
+        description="The code created and maintained for postal purposes to identify a subdivision of addresses and "
+        "postal delivery points.",
+        example="75000",
+        max_length=10,
+    )
+    post_name: Optional[str] = Field(
+        None,
+        title="Post name",
+        description="A name created and maintained for postal purposes to identify a subdivision of addresses and "
+        "postal delivery points. Usually a city.",
+        example="Paris",
+        max_length=40,
+    )
+    po_box: Optional[str] = Field(
+        None,
+        title="PO box",
+        description="A location designator for a postal delivery point at a post office, usually a number.",
+        example="9383",
+        max_length=10,
+    )
+    admin_unit_level1: Optional[ISO_3166_1_Alpha_3] = Field(
+        None,
+        title="Admin unit level 1",
+        description="The name of the uppermost level of the address, almost always a country. ISO 3166 three "
+        "character (Alpha 3) format",
```

Comment:
Let's add the period here as well since it is in the other address fields as well. Updated in the spreadsheet as well.